src/state/slices/tradeQuoteSlice/selectors.ts

```diff
@@ -480,25 +418,25 @@ export const selectBuyAmountAfterFeesUserCurrency = createSelector(
 
 export const selectBuyAmountBeforeFeesUserCurrency = createSelector(
   selectBuyAmountBeforeFeesCryptoPrecision,
-  selectBuyAssetUserCurrencyRate,
+  selectInputBuyAssetUserCurrencyRate,
   (buyAmountBeforeFeesCryptoPrecision, buyAssetUserCurrencyRate) => {
     if (!buyAmountBeforeFeesCryptoPrecision || !buyAssetUserCurrencyRate) return
     return bn(buyAmountBeforeFeesCryptoPrecision).times(buyAssetUserCurrencyRate).toFixed()
   },
 )
 
-export const selectSellAmountUsd = createSelector(
-  selectSellAmountCryptoPrecision,
-  selectSellAssetUsdRate,
+export const selectQuoteSellAmountUsd = createSelector(
+  selectQuoteSellAmountCryptoPrecision,
+  selectInputSellAssetUsdRate,
   (sellAmountCryptoPrecision, sellAssetUsdRate) => {
```

Comment:
Can reflect the vernacular input/quote in the callback if we want to, to signify the intent that this *is* intended and not a mistake?